src/components/ChoosePlan.tsx

```diff
@@ -41,7 +20,7 @@ const ChoosePlan = () => {
         <span className='text-orange-400'>7 days free trial</span>
       </h2>
       <ul className=' w-86 my-2'>
-        {plansData.map(
+        {PLANS.map(
           ({ name, period, price, discountPrice, oldPrice, saveAmount }) => {
             return (
```

Comment:
do you really need return statement here? check arrow functions logic and when return statement is necessary